code/modules/surgery/organs/lungs.dm

```diff
@@ -311,6 +311,35 @@
 		else
 			SEND_SIGNAL(owner, COMSIG_CLEAR_MOOD_EVENT, "smell")
 
+		//NUCLEIUM added. Waste Gas from RBMK Nuclear Reactor	//Monkestation Edit
+		var/nucleium_pp = PP(breath, GAS_NUCLEIUM)
```

Comment:
Add an if check to see if there even is any nucleium, otherwise it will be going through this with every single breath.

Reply:
Interesting that it was like that in the NSV13 code base, I'll fix that